docs/csharp/whats-new/index.md

```diff
@@ -1,4 +0,0 @@
----
-redirect_url: csharp-8
-ms.custom: "updateeachrelease"
```

Comment:
It looks it's by design: that way `redirect_url` for the following address:
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new

can be updated when the next version of C# is released.

Reply:
Not sure if this should be left or not. I wonder if both these redirect methods use the same HTTP redirect method, or if this stub file version is less "sticky" like a meta redirect rather than an HTTP 301(Permanent). Probably something someone on your engineering team would need to answer, maybe @dendeli-msft

Reply:
Yes, it's the same redirection method. The difference is that this one we can have the extra metadata to remind ourselves to update the link since we like having hackable urls. You'll see more of that with an upcoming change to the TOC behavior, I believe.

Reply:
@mairaw OK, maybe you can open an issue in the docs-authoring-pack repo on a way to suppress the cleanup for some files. I think it would be easier for you to explain than me

Reply:
what script are you using? I haven't even discovered that one yet 😄

Reply:
It's these ones https://github.com/Microsoft/vscode-docs-authoring/tree/develop/docs-markdown#cleanup-scripts.
You need to open a markdown file before the command pallet in VS code will show the "Docs: Cleanup..." option. Then they have sub-scripts

Reply:
This redirect should stay. As a general guideline, I would think that any redirect *.md files that include `ms.custom: updateeachrelease` should remain.